server/views/components/wip/fields.njk

```diff
@@ -0,0 +1,15 @@
+{% from "components/wip/field.njk" import renderField %}
+
+{% macro renderFields(options, values, errors) %}
+
+  {% for fieldName, field in options.fields %}
+    {# NB:
+      - dependent fields are currently only used in conditionals so are rendered separately
+      - fields starting with _ are internally generated
+    #}
+    {% if not field.dependent and not fieldName.startsWith("_") %}
```

Comment:
what's an internally generate field 🤔? Currently comment/date fields are just ignored because of the `dependant` property not being null (as in this case here)

Reply:
the “time” field is split into hours and minutes so in reality it is 2 fields, but we cannot count those in the loop as the parent is what drives the template. this is roughly how the hmpo library does it too.